site/content/en/docs/faq/_index.md

```diff
@@ -165,3 +165,13 @@ minikube start
 Currently a static IP can only be set when using the Docker or Podman driver.
 
 For more details see the [static IP tutorial]({{< ref "docs/tutorials/static_ip.md" >}}).
+
+## How to ignore the kubeadm requirements and pre-flight checks (such as CPUs numbers) ?
+
+KubeAdm has several software/hardware requirements to ensure a healthy Kubernetes cluster, to ignore all of them,
+(For example if you want to run minikube on 1 CPU, you can run:
+```
+minikube start --force --extra-config=kubeadm.skip-phases=preflight
+```
+note that, this is not recommended but might be the only way for some users who are okay with performance/stablity side-effects.
```

Comment:
```suggestion
This is not recommended, but for some users who are willing to accept potential performance or stability issues, this may be the only option.
```